spec/services/medication_dispensation_service_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe MedicationDispensationService, type: :model do
+  it "returns a hash of hashes containing key/value pairs of attributes of AppointmentsScheduledDaysDistribution for a region and a period range" do
```

Comment:
Alternative
```
returns bucketed days of medications data
```

This can be written more declaratively by constructing a hash with values and comparing directly.